### PR TITLE
Local storage for payloads

### PR DESCRIPTION
**Pull request by lemaitre-aneo**
This PR adds a new adapter to store payloads in a local filesystem. This is intended to replace Redis in case of local deployment.

### CODE REVIEW

**Comment by lemaitre-aneo** [comment]
With (almost) empty payloads and results, `LocalStorage` is consistently a tiny bit faster than `Redis` (~4%).

File: Adaptors/LocalStorage/src/ObjectStorage.cs [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    {
+      // We need to await the read task before starting to read another chunk
+      var read       = await readTask.ConfigureAwait(false);
+      var readBuffer = buffer;
+
+      if (read == 0)
+      {
+        yield break;
+      }
+
```
Comment (aneojgurhem) [inline Adaptors/LocalStorage/src/ObjectStorage.cs:135]: you should be able to put that in the while

Comment (lemaitre-aneo) [reply Adaptors/LocalStorage/src/ObjectStorage.cs:135]: I don't a see way to do it without duping most of the loop body.

Comment (aneojgurhem) [reply Adaptors/LocalStorage/src/ObjectStorage.cs:135]:
```suggestion
    int read;
    while ((read = await readTask.ConfigureAwait(false)) == 0)
    {
      var readBuffer = buffer;

```

Comment (aneojgurhem) [reply Adaptors/LocalStorage/src/ObjectStorage.cs:135]: Moreover, you can do things after the `yield return` but before the end of the while block.

File: Control/Submitter/src/ArmoniK.Core.Control.Submitter.csproj [hunk 2]
```diff
@@ -29,23 +29,24 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore.Web" Version="2.48.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.17.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="1.3.1" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc9.6" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc9.6" />
```
Comment (aneojgurhem) [inline Control/Submitter/src/ArmoniK.Core.Control.Submitter.csproj:36]: This means that the pipeline does not check if the csproj files are correctly formatted

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**